addons/google_calendar/models/google_credentials.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import requests
+from odoo.addons.google_calendar.models.google_sync import google_calendar_token
+from datetime import timedelta
+
+
+from odoo import fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons.google_account.models.google_service import GOOGLE_TOKEN_ENDPOINT
+from odoo.addons.google_calendar.utils.google_calendar import GoogleCalendarService, InvalidSyncToken
+
+_logger = logging.getLogger(__name__)
+
+class GoogleCredentials(models.Model):
+    """"Google Account of res_users"""
+
+    _name = 'google.calendar.credentials'
+    _description = 'Google Calendar Account Data'
+
+    user_ids = fields.One2many('res.users', 'google_cal_account_id', required=True)
+    calendar_rtoken = fields.Char('Refresh Token', copy=False, groups="base.group_system")
+    calendar_token = fields.Char('User token', copy=False, groups="base.group_system")
+    calendar_token_validity = fields.Datetime('Token Validity', copy=False)
+    calendar_sync_token = fields.Char('Next Sync Token', copy=False)
```

Comment:
no group?

Reply:
> no group? (perhaps it's a good idea to add it?)

I am not sure but doesn't the calendar_token_validity and calendar_sync_token , should be readable for regular users in order to make the google request?